screens/createNewPassword.js

```diff
@@ -0,0 +1,63 @@
+import { Image, View } from "react-native";
+import { useTailwind } from "tailwind-rn";
+import { IconInput } from "../components/inputs";
+import { RoundButton } from "../components/buttons";
+
+const CreateNewPassword = ({ route }) => {
+  const tailwind = useTailwind();
+
+  const updatePassword = async () => {
+    await fetch("http://localhost:8000/api/auth/resetAccountPassword", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        passwordResetToken: route.params.token,
+        newPassword: "newPassword",
```

Comment:
This password should be dynamic, not hard coded (it should be the user's new password).